frontend/packages/dev-console/src/components/project-access/ProjectAccessForm.tsx

```diff
@@ -0,0 +1,185 @@
+import * as React from 'react';
+import * as _ from 'lodash';
+import { Formik } from 'formik';
+import { k8sCreate, k8sKill, k8sPatch } from '@console/internal/module/k8s';
+import { RoleBindingModel } from '@console/internal/models';
+import { LoadingBox, PageHeading } from '@console/internal/components/utils';
+import {
+  validationSchema,
+  UserRoleBinding,
+  filterRoleBindings,
+  getRolesWithNameChange,
+  getNewRoles,
+  getRemovedRoles,
+  getFinalRoles,
+  generateRoleBindingName,
+  Verb,
+  Roles,
+  getUserRoleBindings,
+} from './projectAccessForm-utils';
+
+export interface ProjectAccessFormProps {
+  ProjectAccessFormComponent: React.ComponentType<any>;
```

Comment:
You do not need to pass the `FormComponent` like this from the parent as a prop in this case. It was done this way for import forms because there were multiple form components that were being loaded dynamically based on the parameters. In your case its just one component so its better to remove this complexity.